src/components/KDateTimePicker/KDateTimePicker.vue

```diff
@@ -545,7 +554,7 @@ export default defineComponent({
 @import '@/styles/functions';
 @import '@/styles/mixins';
 
-$timepicker-min-width: 360px;
+$timepicker-min-width: 400px;
```

Comment:
non-blocking: could this have an impact on smaller screens or the component overflowing its parent?

Reply:
Reverting, i can always override min-width in khcp-ui.